### PR TITLE
Mobile first complete

### PR DESCRIPTION
**Pull request by migcm06**
In this part of the exercise, I worked on the section on recent projects. 

I created cards for each project and also coded the "About Myself" section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +35,128 @@ <h2>I'm a Software Developer</h2>
...
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="techs-card">
+                    <li class="tech-card">HTML</li>
+                    <li class="tech-card">Bootstrap</li>
+                    <li class="tech-card">Ruby</li>
+                </ul>
+                <button class="c-project">See project</button>
+            </div>
```
Comment (algerina) [inline index.html:80]: - [x] It is required that for each project the content is aligned with the flexbox. Kindly use flexbox to align the content of each card.

File: styles.css [hunk 2]
```diff
@@ -94,3 +86,279 @@ ul.social {
...
+
+.about-me {
+  background-color: #1c1a19;
+  background-image: url(./images/Illustration2.png);
+  background-repeat: no-repeat;
+  background-position: 99% 21%;
+  height: 300vh;
+  margin-bottom: 31%;
+}
+
```
Comment (algerina) [inline styles.css]: - [x] Same requirement here, it is recommended to use flexbox to align the content of the section.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @migcm06 ,

Great job so far! 

May I highlight the following: 
- [x] Your work is clean and well organized👏  ✔️ 
- [x] Your project looks awesome✔️ 




## CHANGES REQUIRED:muscle::clap:
Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: styles.css [hunk 3]
```diff
@@ -94,3 +86,283 @@ ul.social {
...
+  padding: 4%;
+  border: none;
+}
+
+.card {
+  background-image: url(./images/card.png);
+  background-repeat: no-repeat;
+  background-size: 100% 105%;
+  width: auto;
+  height: 60vh;
```
Comment (rica213) [inline styles.css]: - [x] [OPTIONAL] Kindly set the **height** to **auto**, it will prevent your content to  overflow 🌊 🌊

File: styles.css [hunk 4]
```diff
@@ -94,3 +86,283 @@ ul.social {
...
+.last-card {
+  margin-bottom: 20%;
+}
+
+.about-me {
+  background-color: #1c1a19;
+  background-image: url(./images/Illustration2.png);
+  background-repeat: no-repeat;
+  background-position: 99% 21%;
+  height: 300vh;
```
Comment (rica213) [inline styles.css]: - [x] [OPTIONAL] As I mentioned above, kindly set the height to **auto**, it will prevent your content to overflow 🌵 🚿

**Review by rica213 — APPROVED** [review]
# 🟢 STATUS: APPROVED
Hi Miguel, 
My name is Larissa. I am your code reviewer 🥰

Your project is complete!  You have implemented all the required changes requested by the reviewers ❤️  I add some **OPTIONAL** changes to improve your code. Kindly implement them before merging your branch 🛸 🛸 
That is all! Congratulations! 🎉 



## Highlights

- [ ] Use of semantic HTML ✔️ 
- [ ]  Organized an clean code ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (migcm06) [reply styles.css]: Thanks Larissa

**Pull request merged**